docker_images/mindspore/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM tiangolo/uvicorn-gunicorn:python3.8
+LABEL maintainer="me <me@example.com>"
+
+# Add any system dependency here
+RUN apt-get update -y && apt-get install libglib2.0-dev libsm6 libxrender1 libgl1-mesa-glx -y
+
+COPY requirements.txt /app
+RUN /usr/local/bin/python -m pip install --upgrade pip && \
+    pip install --no-cache-dir -r requirements.txt && \
+    pip install --no-cache-dir git+https://github.com/tinyms-ai/tinyms
```

Comment:
Can we move this to `requirements.txt`  ? it does work on github installs too.

Reply:
Yes, I will update it.